apps/chat/src/store/application/application.epics.ts

```diff
@@ -74,23 +78,30 @@ const updateApplicationEpic: AppEpic = (action$) =>
   action$.pipe(
     filter(ApplicationActions.update.match),
     switchMap(({ payload }) => {
-      const newApplicationId = ApiUtils.encodeApiUrl(
-        getGeneratedApplicationId(payload.applicationData),
-      );
-      if (payload.oldApplicationId !== newApplicationId) {
-        return ApplicationService.move({
-          sourceUrl: payload.oldApplicationId,
-          destinationUrl: newApplicationId,
-          overwrite: false,
-        }).pipe(
-          switchMap(() => of(ApplicationActions.edit(payload.applicationData))),
-          catchError((err) => {
-            console.error('Failed to update application:', err);
-            return of(ApplicationActions.updateFail());
-          }),
-        );
+      const newPayload = regenerateApplicationId(
```

Comment:
lets give another name to this var. Actually we destructured payload here.
It could be something like `updatedCustomApplication`